Sources/SwiftSyntax/LazyNonThreadSafeCache.swift

```diff
@@ -12,12 +12,15 @@
 
 import Foundation
 
-/// AtomicCache is a wrapper class around an uninitialized value.
+/// LazyNonThreadSafeCache is a wrapper class around an uninitialized value.
 /// It takes a closure that it will use to create the value atomically. The
```

Comment:
Nitpick: Can you also remove most of the guarantees from this comment?

Reply:
Sure, comments updated.